### ISSUE TITLE
build command tries to build lots of things?

### ISSUE THREAD

**Issue by rcorces** [databio/refgenie#67]
Full disclosure, I havent used refgenie since the very beginning and it appears quite updated so this is my first time really using refgenie. I'm using a custom genome (actually an N-masked version of hg38). When I type `refgenie build --input hg38_Nmask.fa --asset bowtie2` it does build the bowtie2 index but then also goes and tries to build hisat2, then kallisto, then who knows what. This ultimately throws errors because I dont have those packages loaded.

I know the docs page says the further documentation on building specific assets is forthcoming. So maybe I'm doing something wrong here.

**Comment by nsheff** [comment]
TL;DR: give me a day or two and it will be working...

you are not doing anything wrong... you've caught me right in the middle of revamping the build system... https://github.com/databio/refgenie/issues/66

So the docs don't quite exactly match the functionality yet...

on the 'buildflex' branch the behavior is different, and only individual assets are built. But I just wrote that, like, today.

as of right now on the master branch the way to toggle asset builds is that you have to pass a [refgenie.yaml configuration file](https://github.com/databio/refgenie/blob/b4bab682d7f753387ca90f6041516836894f511c/refgenie/refgenie.yaml)  file. Unfortuantely, that is [broken](https://github.com/databio/refgenie/issues/65). So, you'll need to wait for me to make a new release and then it will work as the docs say...

**Comment by rcorces** [comment]
Ok. No rush on my end. It built what I needed it to build. Close this whenever you're ready to do so.

**Comment by nsheff** [comment]
the release version `build` function should work better now.
